tests/rules/space-before-brace.js

```diff
@@ -2,9 +2,12 @@
 
 var lint = require('./_lint');
 
-var file = lint.file('space-before-brace.scss');
+//////////////////////////////
```

Comment:
Other rules that don't apply to Sass skip silently and I think we should stick to that in case someone has a mix of sass and sass

> On Oct 7, 2015, at 5:27 PM, Dan Purdy <notifications@github.com> wrote:
> 
> In tests/rules/space-before-brace.js:
> 
> > @@ -2,9 +2,12 @@
> >  
> >  var lint = require('./_lint');
> >  
> > -var file = lint.file('space-before-brace.scss');
> > +//////////////////////////////
> I feel this rule should probably provide a warning itself if enabled on a Sass syntax file? What do you reckon?
> 
> —
> Reply to this email directly or view it on GitHub.
>